README.md

```diff
@@ -73,14 +73,18 @@ To clear the local history of updates:
 Other frontends
 -------------------
 
-Currently [GNOME Software](https://wiki.gnome.org/Apps/Software) is the only graphical
-frontend available.  When compiled with firmware support, it will check for updates
-periodically and automatically download firmware in the background.
-
-After the firmware has been downloaded a popup will be displayed in Gnome Software
-to perform the update.
-
-On Dell IoT gateways, [Wyse Cloud Client Manager (CCM)](http://www.dell.com/us/business/p/wyse-cloud-client-manager/pd)
-has been built with fwupd support.
-The remote administration interface can be used to download and deploy
-firmware updates.
+ 1. [GNOME Software](https://wiki.gnome.org/Apps/Software) is the graphical
+ frontend available, When compiled with firmware support, it will check for
```

Comment:
You changed a comma to a period here so it's now a run-on sentence.  Can you please change that back?